### PR TITLE
bump deps

### PR DESCRIPTION
**Pull request by odsod**
Fix lint error from deprecated `grpc.WithInsecure()`.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -27,8 +27,8 @@ spanner-generate:
 .PHONY: go-mod-tidy
 go-mod-tidy:
 	$(info [$@] tidying Go module files...)
-	@go mod tidy
-	@cd cmd/iamctl && go mod tidy
+	@go mod tidy -compat=1.17
+	@cd cmd/iamctl && go mod tidy -compat=1.17
```
Comment (ericwenn) [inline Makefile:31]: why is this needed? looks like both modules are 1.17 already?

Comment (odsod) [reply Makefile:31]: antlr seems to have introduced a new module `github.com/antlr/antlr4/runtime/Go/antlr` which provides the same package as `github.com/antlr/antlr4`. `go mod tidy` only works in 1.17 mode, but now I managed to move the dependency over to the new module completely, and then it works without it.

**Review by ericwenn — APPROVED** [review]
